x-pack/plugins/ml/public/jobs/jobs_list/components/start_datafeed_modal/start_datafeed_modal.js

```diff
@@ -91,13 +94,20 @@ export class StartDatafeedModal extends Component {
       showCreateWatchFlyout,
       allowCreateWatch,
       createWatch: false,
+      now,
     });
   }
 
   save = () => {
     const { jobs } = this.state;
-    const start = moment.isMoment(this.state.startTime) ? this.state.startTime.valueOf() : this.state.startTime;
+    let start = moment.isMoment(this.state.startTime) ? this.state.startTime.valueOf() : this.state.startTime;
     const end = moment.isMoment(this.state.endTime) ? this.state.endTime.valueOf() : this.state.endTime;
+    if (start === end) {
```

Comment:
I wonder if it might be better to disable the start button if the user has picked the same start and end times? Otherwise the 'hidden' behaviour of moving the start time back by 1ms results in a job run with just 1ms worth of data, which won't be of any use.

Reply:
ok, yeah i agree. change coming...